src/Hosting.Services.Web/Middlewares/ObsoleteCorrelationHeadersMiddleware.cs

```diff
@@ -113,6 +113,7 @@ private void ExtractCorrelationFromRequest(HttpRequest request)
 
 		private static readonly string[] s_transactionsNames = {
 			"corrtid",					// Correlation transaction query parameter
+   			"correlationId",
```

Comment:
Do you think you should have it in the array below on line 121 instead? This is for transaction

Reply:
for the use case we have it's just parameter being sent, not in the headers

Reply:
fixed